lib/index.js

```diff
@@ -816,12 +668,12 @@ function _getAdminCredentials(callback) {
 }
 
 function _getAddUserOptions() {
-  const local = bedrock.config.mongodb.local;
+  const config = bedrock.config.mongodb;
   return bedrock.util.extend({}, bedrock.config.mongodb.writeOptions, {
     roles: [
       'dbOwner',
-      {role: 'dbAdmin', db: local.name, collection: local.collection},
-      {role: 'readWrite', db: local.name, collection: local.collection}
+      {role: 'dbAdmin', db: config.name, collection: config.collection},
+      {role: 'readWrite', db: config.name, collection: config.collection}
```

Comment:
I believe this pertains to your comments in the description of the PR.  This helper function formerly only transacted with the local database which is not a thing anymore.  Is there a reason why the function would now be altering the regular, non-local database?  Seems to me that this helper function just goes away.

Reply:
just the helper or `_createUser` entirely?

Reply:
I think there is a cascade of code that gets removed with the local db.  The call to _createUser uses db param that is no longer set, so that function probably goes away too.  If that gets removed, then _getAdminCredentials is gone, and the prompt dep, and _getAddUserOptions.  Maybe other code too.

Reply:
@davidlehn cascade of code removed here: https://github.com/digitalbazaar/bedrock-mongodb/pull/52/commits/7fa8314db345860c3e1441673691666fd252f6bd @mattcollier removed `_createUser` and related code.